libraries/AP_HAL_ESP32/Scheduler.cpp

```diff
@@ -571,8 +559,11 @@ void IRAM_ATTR Scheduler::_main_thread(void *arg)
 
     sched->set_system_initialized();
 
+    //initialize WTD for current thread on FASTCPU, all cores will be (1 << CONFIG_FREERTOS_NUMBER_OF_CORES) - 1
```

Comment:
the code used to be explicit about the fact that it was disabling WDT on both core/s with disableCore0WDT/disableCore1WDT, and now that you've simplified this, it appears wdt is now enabled on one core, and disabled on the other, which is fine, if it works, but perhaps a bit more of a warning for those coming after you in the comments might help here.

Reply:
The issue was that the new Watchdog Timer (WDT) driver did not handle the disabling of the WDT on specific cores well, resulting in the console being flooded with errors about missing WDT initialization. Additionally, I was unable to disable the IDLE task, which made things more complicated. As a result, I decided that explicitly initializing the WDT at the end was a simpler solution, and it worked.